test/kubemark/resources/start-kubemark-master.sh

```diff
@@ -421,6 +582,15 @@ function start-kubemaster-component() {
 		sed -i -e "s@{{etcd_image}}@${ETCD_IMAGE}@g" "${src_file}"
 	elif [ "${component}" == "kube-addon-manager" ]; then
 		setup-addon-manifests "addons" "kubemark-rbac-bindings"
+	elif [ "${component}" == "kube-apiserver" ]; then
+		local audit_policy_config_mount=""
+		local audit_policy_config_volume=""
+		if [[ "${ENABLE_APISERVER_ADVANCED_AUDIT:-}" == "true" ]]; then
+			audit_policy_config_mount=$"- name: auditpolicyconfigmount\n      mountPath: ${audit_policy_file}\n      readOnly: true"
+			audit_policy_config_volume=$"- name: auditpolicyconfigmount\n    hostPath:\n      path: ${audit_policy_file}\n      type: FileOrCreate"
```

Comment:
I think it's better to have it as a multiline message, e.g.

```
read -d '' audit_policy_config_volume << EOF
    - name: auditpolicyconfigmount
      mountPath: ${audit_policy_file}
      readOnly: true
EOF
```

But that's up to you

Reply:
Cool suggestion - changed it. Though it looks at bit off aesthetically:)